client/src/components/LogIn.tsx

```diff
@@ -27,16 +27,37 @@ const theme = createTheme();
 
 const LogIn = (props: HookProps) => {
   let navigate = useNavigate();
-  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+
+  const [email, setEmail] = React.useState("");
+  const [password, setPassword] = React.useState("");
+  const [message, setMessage] = React.useState("");
+
+  const handleSubmit = async (event: React.FormEvent<HTMLFormElement>) => {
     event.preventDefault();
-    const data = new FormData(event.currentTarget);
-    // Update the modal state 
-    props.setState(false);
-    // eslint-disable-next-line no-console
-    console.log({
-      email: data.get('email'),
-      password: data.get('password'),
-    });
+    try {
+      // set the login url
+      let res = await fetch("http://127.0.0.1:3000/login", {
+        method: "POST",
+        headers: {
+          'Content-Type': 'application/json'
+        },
+        body: JSON.stringify({
```

Comment:
You should be able to send as a json object, don't stringify if not necessary

Reply:
Tried to remove it but it raises type errors, it seems like `stringify()` is required to parse the JSON data in a POST request (https://upmostly.com/tutorials/how-to-post-requests-react#:~:text=Finally%2C%20we%20specify,its%20original%20form). Please let me know if there's better solution.